### PR TITLE
document backend standardization role

### PR DESCRIPTION
**Pull request by tiagojsag**

### CODE REVIEW

File: _docs/backenders/strategy-2021/roles/standardization.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# Standardization role definition
+
+**Current responsible**: Tiago Garcia
+
+## Responsibilities
+
+- Improve technical standardization within our FA
+    
+  - Identify commonly or recurring tasks/challenges/problems
```
Comment (hotzevzl) [inline _docs/backenders/strategy-2021/roles/standardization.md:16]:
```suggestion
  - Identify common or recurring tasks/challenges/problems
```

File: _docs/backenders/strategy-2021/roles/standardization.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- Less time spent "reinventing the wheel" (for all FAs involved)
+- More consensual decisions documented by all on the FA wiki
+- More efficient development process overall (for all FAs involved)
+- Better synergies across FAs
+
+## Decision-making
+
+- Which standardization topics to bring up for discussion/work
+- Representation of our FA on company-wide standardization initiatives
+  - Autonomy to make small decisions
```
Comment (hotzevzl) [inline _docs/backenders/strategy-2021/roles/standardization.md:38]: PHP ALL THE THINGS!!!!!11! 😉

**Review by hotzevzl — APPROVED** [review]

File: _docs/backenders/strategy-2021/roles/standardization.md (hunk 2 above)
Comment (tiagojsag) [reply _docs/backenders/strategy-2021/roles/standardization.md:38]: !!!!!!!111oneeleven

File: _docs/backenders/strategy-2021/roles/standardization.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Template
```
Comment (ikas) [inline _docs/backenders/strategy-2021/roles/standardization.md:3]:
```suggestion
title: Standardization role definition
```

**Review by ikas — APPROVED** [review]
Minor tweak, otherwise LGTM :rocket:

**Review by develaper — APPROVED** [review]

**Review by aagm — APPROVED** [review]

**Review by agnessa — APPROVED** [review]

**Review by alexeh — APPROVED** [review]

**Pull request merged**